google-cloud-logging/src/test/java/com/google/cloud/logging/BaseSystemTest.java

```diff
@@ -468,12 +468,14 @@ public void testLoggingHandler() throws InterruptedException {
     logger.addHandler(handler);
     logger.setLevel(Level.INFO);
     logger.info("Message");
+    String filter =
+            "logName = projects/" + logging().getOptions().getProjectId() + "/logs/" + logName;
```

Comment:
Use https://github.com/garrettjonesgoogle/api-client-staging/blob/master/generated/java/proto-google-cloud-logging-v2/src/main/java/com/google/logging/v2/LogName.java#L26 instead of composing the String manually.